firestore/src/main/java/com/firebase/ui/firestore/paging/FirestorePagingAdapter.java

```diff
@@ -34,6 +34,14 @@
     private final LiveData<LoadingState> mLoadingState;
     private final LiveData<FirestoreDataSource> mDataSource;
 
+
+    private final Observer<FirestoreDataSource> mDataSourceObserver = new Observer<FirestoreDataSource>() {
+        @Override
+        public void onChanged(@Nullable FirestoreDataSource source) {
+
```

Comment:
Can we add a comment inside here explaining why we need an empty observer so that future people who read this code know?

Reply:
@samtstern yes.
What comment should be added ?
Please suggest me.

Reply:
`//Observer to hold the value for FirestoreDataSource` ??